src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentPlanCard/CurrentPlanCard.spec.tsx

```diff
@@ -62,7 +65,7 @@ const queryClient = new QueryClient({
   defaultOptions: { queries: { retry: false } },
 })
 
-const wrapper = ({ children }) => (
+const wrapper = ({ children }: { children: React.ReactNode }) => (
```

Comment:
I should've noted this in my other comments, but we typically go with `React.FC` off of the wrapper or the factory function over directly typing children in this way.

Reply:
No prob! I actually went ahead and migrated over every instance we had in the repo here: [d477ef0](https://github.com/codecov/gazebo/pull/2642/commits/d477ef0f37c44093d1ded428827afdf424852b21)

Lmk if you think this is something we should look to add a lint rule over